Update readme.md

santip21 opened the pull request:
## 🎯 Objective

Add something to drink addressing Angus allergies to oranges


## 🛠️ Changes made

Added apple juice to the recipe


## 👥 Who needs to review this Pull Request? 

@caitrin-hall @AngusMaguir3

santip21 committed:
Update readme.md

caitrin-hall commented:
Hey, apple juice has way too much sugar. What about grapefruit juice instead? Healthy AND delicious!

caitrin-hall committed:
Update readme.md

Clarified that the degrees are in Fahrenheit, and can we take out the cinnamon because I'm allergic! :)

AngusMaguir3 reviewed (approved):
Recipe may need to be refactored later but it looks good to me.

caitrin-hall reviewed (approved):
looks awesome
